New illegal outfits, slave and drugs (#7181)
endless-sky/endless-sky: data/human/outfits.txt

	"unplunderable" 1
	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."

> outfit "Drugs"

Reviewer: I  would prefer if the name is a bit more specific than just "Drugs".

Reviewer 2: Would propose "Combat Anesthetic" or "Combat Stimulants" though those two classes of drugs tend to have opposite effects of eachother. Maybe avoid beating around the bush and call them "Combat Methamphetamines"?

	plural "Drugs"
	category "Hand to Hand"
	cost 10000

> 	thumbnail "outfit/Drugs"

Reviewer: ```suggestion
	thumbnail "outfit/drugs"
```

Afaik file name should be all lowercase. So apply this change to the file name as well.

	"capture attack"  2.2
	"capture defense" 1.3
	"illegal" 800000
	"unplunderable" 1

> 	description " Although they decrease your crew's jugment and are illegal, drugs can increase your crew's fighting spirit and will numb them to a certain amount of pain."

Reviewer: ```suggestion
	description "Although they decrease your crew's jugment and are illegal, drugs can increase your crew's fighting spirit and will numb them to a certain amount of pain."
```

Reviewer 3: ```suggestion
	description " Although they decrease your crew's judgement and are illegal, these narcotics can increase your crew's fighting spirit and will numb a certain amount of pain."
```

> 	description " Although they decrease your crew's jugment and are illegal, drugs can increase your crew's fighting spirit and will numb them to a certain amount of pain."

Reviewer 2: Surely a drug like this would be severely addictive and destructive - look at what happened to Nazis who consumed Panzerschokoloade. Would propose adding consequence events rather than letting First Last get off scot-free.

Reviewer 4: Panzerschokolade was just meth iirc, and not just Germans did it - the Allies also did distribute meth to soldiers.

But seconding you on consequences of some sort. Maybe the crew experiences severe flashbacks and PTSD or end up completely divorced from reality? The latter happened to a USA soldier who volunteered for combat drug experiments around 1970...

> 	description "Although they decrease your crew's judgement and are illegal, these narcotics can increase your crew's fighting spirit and will numb a certain amount of pain."

Reviewer 2: What does "decrease your crew's judgement" mean? Do they lower your crew's critical thinking skills, or lower their opinion of First Last, or both?

Also, 2.2 attack/1.3 defense is pretty powerful - almost as offensive as Nerve Gas, and almost as defensive as a Protection Station, without requiring any extra outfit space. This item needs some major consequences to not be crazy OP. My first thought would be to add a mission chain which starts out like Bad Trip and ends with First Last overdosing, but that is probably too dark for this game.

It would be great if we could programmatically remove crew with events, but I don't think we can - I feel a sufficient consequence would be a crewmember dying after launch, on a smaller ship possibly majorly impacting their ability to maneuver, and hopefully forcing the player to reconsider their decision to drug up their crew.

Reviewer 4: "What does "decrease your crew's judgement" mean? Do they lower your crew's critical thinking skills, or lower their opinion of First Last, or both?"

Never saw "judgement" used for reputation/opinion, so I assume it means reasoning/thinking skills.

Author: @AbsolutelyLudicrous I had tried to figure out how code missions with crew members, but from what I can see there is no part of the code that will allow me to do it.

I should probably change judgement to perception.

	"map" 12
	description "This data chip contains complete information on the twelve star systems closest to this one: planets, ports, governments, trade prices, available services, etc. You can get all the same information just by exploring those systems yourself, but having a map can save you from making wrong turns if you are trying to travel through new territory to reach a certain system quickly."

outfit "Slave"
	category "Special"
	cost 25000
	thumbnail "outfit/slave"
	"required crew" -1
	"outfit space" -1
	"atrocity" 1
	"unplunderable" 1

> 	description "Slaves are highly illegal but can be a great asset if you are going on a long journey into the middle of nowhere. They take the place of a crew member, and you don't have to pay them a daily wage. They also come with a small bed and space to sleep in."

Reviewer 2: I worked out what bothers me about this description - it describes the slave as an object, a thing. You're not treating the human with anywhere near the amount if sympathy they deserve. The description needs to focus on the cruelty of the person trying to sell you another person, and on the inhumanity of any captain who buys one. This current description makes a human out to be a piece of machinery, something with pros and cons to installing on your ship; it's dehumanizing.

	"map" 12
	description "This data chip contains complete information on the twelve star systems closest to this one: planets, ports, governments, trade prices, available services, etc. You can get all the same information just by exploring those systems yourself, but having a map can save you from making wrong turns if you are trying to travel through new territory to reach a certain system quickly."

outfit "Slave"

> 	plural "Slaves"

Reviewer: ```suggestion

```
Unnecessary, the game already automatically add `s` at the end for plural, use `plural` only for special cases where that would be incorrect.

	category "Special"
	cost 1000

> 	thumbnail "outfit/Ball-and-chain"

Reviewer: ```suggestion
	thumbnail "outfit/ball-and-chain"
```
Same case with Drugs earlier.
